src/briefcase/platforms/macOS/app.py

```diff
@@ -45,14 +46,14 @@ def install_app_support_package(self, app: BaseConfig):
         super().install_app_support_package(app)
 
         # keep only Python lib from support package
-        lib_path = str(self.support_path(app).parent / 'Support' / 'Python' / 'Resources' / 'lib')
+        lib_path = os.fsdecode(self.support_path(app).parent / 'Support' / 'Python' / 'Resources' / 'lib')
 
         with tempfile.TemporaryDirectory() as tmpdir:
             self.shutil.move(lib_path, tmpdir)
-            self.shutil.rmtree(str(self.support_path(app)))
+            self.shutil.rmtree(os.fsdecode(self.support_path(app)))
 
-            self.os.makedirs(str(Path(lib_path).parent))
-            self.shutil.move(str(Path(tmpdir) / 'lib'), lib_path)
+            self.os.makedirs(os.fsdecode(Path(lib_path).parent))
+            self.shutil.move(os.fsdecode(Path(tmpdir) / 'lib'), lib_path)
```

Comment:
`shutil.move` only accepts path-like objects from 3.9, I think we should use there `os.fsdecode()` [link](https://docs.python.org/3/library/shutil.html#shutil.move)

Reply:
Well that's annoying :-)

In that case, you're definitely correct including the conversion here. However, it would be worth adding a docstring so future editors know (a) this isn't redundant, and (b) when we (eventually) drop 3.8 support, dropping the conversion is an option as well.

Reply:
docstring added